### PR TITLE
Adds archived plugins to the site

### PR DESCRIPTION
**Pull request by GameDungeon**
Adds a manually controlled plugin list to display as archived.
image

Somone should make this better in the future, but this implementation works for now.

### CODE REVIEW

**Review by naveen521kk — CHANGES_REQUESTED** [review]
Can you remove the changes from `package-lock.json`?

File: src/main.css [hunk 1]
```diff
@@ -64,4 +64,7 @@ body {
 }
 .hover-card:hover {
    transform: scale(1.07);
+}
+.archived {
+  background-color:#cccccc
 }
```
Comment (naveen521kk) [inline src/main.css:70]: Add a new line here.
```suggestion
}

```

**Comment by GameDungeon** [comment]
Fixed

**Review by naveen521kk — APPROVED** [review]
Lgtm

**Pull request merged**